src/prefect/client/schemas/actions.py

```diff
@@ -731,7 +731,7 @@ class GlobalConcurrencyLimitUpdate(ActionBaseModel):
     """Data used by the Prefect REST API to update a global concurrency limit."""
 
     name: Optional[str] = Field(None)
-    limit: Optional[int] = Field(None)
-    active: Optional[bool] = Field(None)
-    active_slots: Optional[int] = Field(None)
-    slot_decay_per_second: Optional[float] = Field(None)
+    limit: Optional[NonNegativeInteger] = Field(None)
+    active: Optional[NonNegativeInteger] = Field(None)
+    active_slots: Optional[NonNegativeInteger] = Field(None)
+    slot_decay_per_second: Optional[NonNegativeFloat] = Field(None)
```

Comment:
We need these self-validating types so that we can ensure validated inputs to the `update_global_concurrency_limit` method in the client